melos.yaml

```diff
@@ -29,9 +31,19 @@ scripts:
     description: Analyze Dart code in demos.
     run: dart analyze demos --fatal-infos
 
+  compile:webworker:
+    description: Compile Javascript web worker distributable
+    exec: dart run powersync_web_worker:compile_webworker
+    packageFilters:
+      scope:
+        - powersync_web_worker
+
+  update:wasm: sh scripts/init_sqlite_wasm.sh
+
   test:
     description: Run tests in a specific package.
-    run: dart test
+    # Tests on Web are disabled for now. They pass locally but there is a network issue on CI
```

Comment:
Is this still accurate?

Reply:
No, this is not accurate. It will be removed. Thanks for spotting!